jdl/converters/jdl-to-json/jdl-to-json-field-converter.ts

```diff
@@ -17,42 +17,45 @@
  * limitations under the License.
  */
 
-const _ = require('lodash');
-const { isBlobType } = require('../../jhipster/field-types');
-const { UNIQUE, REQUIRED } = require('../../jhipster/validations');
-const { formatComment } = require('../../utils/format-utils');
-const { camelCase } = require('../../utils/string-utils');
-const { CommonDBTypes, RelationalOnlyDBTypes, BlobTypes } = require('../../jhipster/field-types');
+import _ from 'lodash';
+import Validations from '../../jhipster/validations';
+import formatComment from '../../utils/format-utils';
+import { camelCase } from '../../utils/string-utils';
+import fieldTypes from '../../jhipster/field-types';
+import JDLObject from '../../models/jdl-object';
+import { Field } from '../types';
+import JDLEntity from '../../models/jdl-entity';
 
-const { ANY_BLOB, BLOB, IMAGE_BLOB, TEXT_BLOB } = CommonDBTypes;
-const { ANY, IMAGE, TEXT } = BlobTypes;
-const { BYTES } = RelationalOnlyDBTypes;
+const { UNIQUE, REQUIRED } = Validations;
 
-module.exports = {
-  convert,
-};
+const { isBlobType } = fieldTypes;
+const { ANY_BLOB, BLOB, IMAGE_BLOB, TEXT_BLOB } = fieldTypes.CommonDBTypes;
+const { ANY, IMAGE, TEXT } = fieldTypes.BlobTypes;
+const { BYTES } = fieldTypes.RelationalOnlyDBTypes;
+
+export default { convert };
```

Comment:
Same suggestion as before, if we have an issue for the imports, there should be one for exports (inlining default exports and regular exports).

Reply:
Same answer as before. :)